src/Chapter2.hs

```diff
@@ -741,8 +753,9 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
-
+contains :: Int -> [[Int]] -> [[Int]]
+contains _ [] = []
```

Comment:
And here as well. All of the standard list functions usually cover cases with the empty list, so you don't need to worry about it 😉